arangod/Aql/ExecutionBlock.cpp

```diff
@@ -54,6 +54,12 @@ static std::string const& stateToString(ExecutionState state) {
   return unknownString;
 }
 
+struct ExecutionBlockTypeHash {
+  size_t operator()(const ::arangodb::aql::ExecutionBlock::Type& value) const {
+    return size_t(value);
```

Comment:
is this a suitable hash function?

Reply:
its basically a class enum which newer compilers don't understand.